edward/models/dirichlet_process.py

```diff
@@ -10,8 +10,9 @@
 
 
 class DirichletProcess(RandomVariable, Distribution):
-  def __init__(self, alpha, base_cls, validate_args=False, allow_nan_stats=True,
-               name="DirichletProcess", value=None, *args, **kwargs):
+  def __init__(
```

Comment:
Ah, got it. It should pass pep8 according to the additional rules we set up, so no need to bother (https://github.com/blei-lab/edward/blob/master/setup.cfg). (the `pep8` python package automatically uses additional rules from setup.cfg; not sure about flake8.)